test/speed-index.js

```diff
@@ -44,6 +44,30 @@ test('visual progress should have 0 and 100 for different images', async t => {
 	t.is(data.frames[1].getProgress(), 100);
 });
 
+test('perceptual progress should use SSIM', async t => {
```

Comment:
do you want to also add a more narrow test of `calculateFrameSimilarity`? we could then manually verify that calculateFrameSimilarity says the same thing as http://darosh.github.io/image-ssim-js/test/browser_test.html :)